build.sbt

```diff
@@ -32,8 +32,8 @@ lazy val coverageSettings: Seq[Setting[_]] = {
     "<empty>",
     "Reverse.*",
     ".*standardError*.*",
-    ".*govuk_wrapper*.*",
-    ".*main_template*.*",
+    ".*GovUkWrapper*.*",
+    ".*MainTemplate*.*",
```

Comment:
These aren't actually needed in the scoverage exclusions as they don't exist in this service